requirements-wally.txt

```diff
@@ -0,0 +1,4 @@
+https://github.com/ElementsProject/libwally-core/releases/download/release_0.7.8/wallycore-0.7.8-cp36-cp36m-linux_x86_64.whl; 'linux' in sys_platform and python_version == '3.6' --hash=sha256:b973fbabf7c30952d6ab5f626ea5231f06200c5f4f35f07bea19308ab87c8162
+https://github.com/ElementsProject/libwally-core/releases/download/release_0.7.8/wallycore-0.7.8-cp37-cp37m-linux_x86_64.whl; 'linux' in sys_platform and python_version == '3.7' --hash=sha256:77bebee316219c4be5069722e9aa08043278934641c7f677627194b4dfeabfd0
+https://github.com/ElementsProject/libwally-core/releases/download/release_0.7.8/wallycore-0.7.8-cp37-cp37m-macosx_10_14_x86_64.whl; 'darwin' in sys_platform and python_version == '3.7' --hash=sha256:bae6656aa86ed1197e636f69905e10987ef4fd3ee8ba18d16e0f62fcd8aaaaf1
+https://github.com/ElementsProject/libwally-core/archive/release_0.7.8.tar.gz --hash=sha256:b36a9e4e7db155d5a435b43c08e74ed9d812b05ea2becce454700a18217c2296
```

Comment:
wally is at 0.7.9 now see https://github.com/greenaddress/garecovery/blob/master/tools/requirements.txt for latest hashes + windows binary.